lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -46,6 +55,10 @@ class DetailsRenderer {
         return this._renderCode(details);
       case 'node':
         return this.renderNode(/** @type {!DetailsRenderer.NodeDetailsJSON} */(details));
+      case 'criticalrequestchain': // eslint-disable-line no-case-declarations
+        const crcRenderer = new CriticalRequestChainRenderer(this._dom, this._templateContext);
```

Comment:
nit: feels weird to create one every iteration of render, thoughts on having a single instance of it in the constructor? the `setTemplateContext` waterfall is gonna get real annoying though